packages/spelunker-web/src/components/core/GameMap/index.jsx

```diff
@@ -32,13 +33,33 @@ const loadTileIndex = async (tileDirectory) => {
   TILE_INDEX[tileDirectory] = tileIndex;
 };
 
+const loadTile = async (tileUrl, tile, done) => {
+  const blp = await loadBlp(tileUrl);
+
+  if (!blp) {
+    done(new Error('invalid blp'), tile);
+    return;
+  }
+
+  drawBlp(blp, tile);
+  done(undefined, tile);
+};
+
 class MinimapTileLayer extends TileLayer {
+  createTile(coords, done) {
+    const tile = document.createElement('canvas');
+    const tileUrl = this.getTileUrl(coords);
+    loadTile(tileUrl, tile, done);
+
+    return tile;
```

Comment:
This looked a bit odd to me at first glance, but I assume we need the `done` callback  as Leaflet does not support async here? Might warrant a comment here at a later point as `loadTile` will be a lost promise (but still draw to the tile).

Reply:
Yeah, Leaflet appears to still prefer callback style async vs Promises. I'll see about cleaning this up in the next PR.